### PR TITLE
Sign In and Sign UP Pages

### PR DESCRIPTION
**Pull request by PritSapariya**

### CODE REVIEW

File: pos-frontend/src/components/SignUp.jsx [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+const SignUp = (props) => {
+
+  const { register , handleSubmit, formState: { errors }, watch, getValues, reset, setFocus, setError } = useForm();
+  const [ message, setMessage ] = useState(initialState);
+  const navigate = useNavigate()
+
+  const onSubmit = () => {
+    const userData = getValues();
+
+    axios.post("http://localhost:5000/api/auth/signup", userData).then((res) => {
```
Comment (sweetasinha) [inline pos-frontend/src/components/SignUp.jsx]: why is the api url here ... move it to a common file

File: pos-frontend/src/components/SignUp.jsx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+        </Snackbar>
+
+
+        <Grid container component="div" sx={{ height: '100vh', width: "100%" }}>
+          <Grid 
+            item 
+            xs={false} 
+            sm={6} 
+            sx={{
+              backgroundImage: "url(/images/login-img1.jpg)",
```
Comment (sweetasinha) [inline pos-frontend/src/components/SignUp.jsx]: make a ts file file from where you can export all images

File: pos-frontend/src/components/SignUp.jsx [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+                      fullWidth
+                      name="confirmPassword"
+                      label="Confirm Password"
+                      type="password"
+                      id="confirmPassword"
+                      {...register("confirmPassword", { 
+                        required: "Confirm password is required",
+                        validate: (val) => {
+                          if(watch('password') !== val) {
+                            return "Password does not match"
```
Comment (sweetasinha) [inline pos-frontend/src/components/SignUp.jsx]: define all messages in a common file and import from there ... change this in every place

File: pos-frontend/src/components/SingIn.jsx [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+      message: ''
+    })
+  }
+  
+
+  const onSubmit = (event) => {
+    
+    const data = getValues();
+    
+    let updatedUrl = url + "/auth/signin"
```
Comment (sweetasinha) [inline pos-frontend/src/components/SingIn.jsx]: use template literals to concatenate string with any dynamic value

File: pos-frontend/src/components/SignUp.jsx (hunk 1 above)
Comment (PritSapariya) [reply pos-frontend/src/components/SignUp.jsx]: Moved to common file and using is as a const

File: pos-frontend/src/components/SignUp.jsx (hunk 2 above)
Comment (PritSapariya) [reply pos-frontend/src/components/SignUp.jsx]: Done

File: pos-frontend/src/components/SignUp.jsx (hunk 3 above)
Comment (PritSapariya) [reply pos-frontend/src/components/SignUp.jsx]: Under the utility folder, validationMessages.js is created.

File: pos-frontend/src/components/SingIn.jsx (hunk 4 above)
Comment (PritSapariya) [reply pos-frontend/src/components/SingIn.jsx]: Completed

**Pull request merged**